packages/@uppy/angular/projects/uppy/angular/tsconfig.lib.json

```diff
@@ -3,6 +3,7 @@
   "extends": "../../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../../out-tsc/lib",
+    "moduleResolution": "bundler",
```

Comment:
Another approach that worked was switching the package to full ESM (moduleResolution: nodenext), but that would have introduced a breaking change.

Reply:
Do the angular docs (or CLI scaffold) show that this is the correct option?